frontend/catalyst/compiler.py

```diff
@@ -407,23 +397,30 @@ def run(self, mlir_module, *args, **kwargs):
             **kwargs,
         )
 
-    def get_output_of(self, pipeline) -> Optional[str]:
+    def get_output_of(self, workspace, pipeline) -> Optional[str]:
```

Comment:
Is workspace unused in this function (besides the checks)?

Reply:
Ah yes, I think this was an attempt to make the compiler functional (i.e., without any state), but I forgot to remove `last_compiler_output`. I'll work on it tomorrow.